Documentation/installation/k8s-install-helm.rst

```diff
@@ -95,7 +95,6 @@ Install Cilium
             --namespace kube-system \\
             --set aksbyocni.enabled=true \\
             --set nodeinit.enabled=true
-         
          .. note::
 
             Installing Cilium via helm is supported only for AKS BYOCNI cluster and 
```

Comment:
This one would be nice to fix, please. You moved the note out of the `parsed-literal` block in your first version, which was a good idea - just have it as a separate commit, please.

Reply:
Okay!